glottolog3/views.py

```diff
@@ -342,40 +343,41 @@ def bp_api_search(request):
         kind = 'Glottocode'
     else:
         # list of criteria to search languoids by
-        crit = [Identifier.type == 'name']
+        filters = []
         ul_iname = func.unaccent(func.lower(Identifier.name))
         ul_name = func.unaccent(term)
         if namequerytype == 'whole':
-            crit.append(ul_iname == ul_name)
+            filters.append(ul_iname == ul_name)
         else:
-            crit.append(ul_iname.contains(ul_name))
+            filters.append(ul_iname.contains(ul_name))
         if not multilingual:
             # restrict to English identifiers
-            crit.append(func.coalesce(Identifier.lang, '').in_((u'', u'eng', u'en')))
-        crit = Language.identifiers.any(and_(*crit))
-        # add ISOs to query if length == 3
-        iso = Languoid.identifiers.any(type=IdentifierType.iso.value, name=term) if len(term) == 3 else None
```

Comment:
are iso identifiers still included?

Reply:
iso's are themselves identifiers. the previous code, would search for isos and names separately for some reason... but i haven't really seen a reason to just let them be grouped in as identifiers, so it's probably something we can leave for now. i'll add a test searching for an iso.